udata/commands/db.py

```diff
@@ -383,3 +386,172 @@ def print_and_save(text: str):
+
+        return None
+
+    with click.progressbar(
+        Dataset.objects,
+        Dataset.objects().count(),
+    ) as datasets:
+        for dataset in datasets:
+            for resource in dataset.resources:
+                if resource.id not in resources:
+                    resources[resource.id] = {"resources": [], "datasets": set()}
+
+                resources[resource.id]["resources"].append(resource)
+                resources[resource.id]["datasets"].add(dataset)
+
+    resources = {id: info for id, info in resources.items() if len(info["resources"]) != 1}
+
+    count_resources = 0
+    count_datasets = 0
+    for id, info in resources.items():
+        if len(info["datasets"]) == 1 and not duplicate_inside_dataset:
+            continue
+
+        if len(info["datasets"]) > 1 and not duplicate_outside_dataset:
+            continue
+
+        # Filter out meteo france
+        if (
+            exclude_meteo_france
+            and list(info["datasets"])[0].organization
+            and str(list(info["datasets"])[0].organization.id) == "534fff8ba3a7292c64a77ed4"
+        ):
+            continue
+
+        # Filter everything except meteo france
+        if only_meteo_france and (
+            not list(info["datasets"])[0].organization
+            or str(list(info["datasets"])[0].organization.id) != "534fff8ba3a7292c64a77ed4"
+        ):
+            continue
+
+        count = len(info["resources"])
+        print(f"With ID {id}: {count} resources")
+        for dataset in info["datasets"]:
+            count_datasets += 1
+            print(f"\t- Dataset#{dataset.id} {dataset.title}")
+        print("")
+        for resource in info["resources"]:
+            count_resources += 1
+            print(
+                f"\t- Resource {resource.title} ({get_checksum_value(resource)} / {resource.url})"
+            )
+
+        print()
+
+        if len(info["datasets"]) == 1 and len(info["resources"]) == 2:
+            dataset = next(iter(info["datasets"]))
+
+            resource1 = info["resources"][0]
+            resource2 = info["resources"][1]
+
+            new_resources = []
+            highlight_ids = [id]
+            if (
+                get_checksum_value(resource1) == get_checksum_value(resource2)
+                and resource1.url == resource2.url
+            ):
+                print(
+                    "Since checksum and URL are the same, fixing by removing the second resource…\n"
+                )
+
+                for r in dataset.resources:
+                    # If it's the duplicated resource we're interested in and
+                    # that ID was already added to the new_resources (so we are
+                    # on the second resource), do not add it.
+                    if r.id == id and id in [r.id for r in new_resources]:
+                        continue
+
+                    new_resources.append(r)
+            else:
+                print(
```

Comment:
Maybe change the wording if `--fix` isn't applied?

Reply:
Add a prefix on all log with the fix / dryrun status dd1a10885